scene/gui/code_edit.cpp

```diff
@@ -3344,5 +3342,5 @@ bool CodeCompletionOptionCompare::operator()(const ScriptLanguage::CodeCompletio
 			return l.matches[i].second > r.matches[i].second;
 		}
 	}
-	return l.display < r.display;
+	return l.display.to_lower() < r.display.to_lower();
```

Comment:
Might be useful to use the existing function:
```suggestion
	return l.display.nocasecmp(r.display) < 0;
```

Reply:
wow didn't know this existed was exactly what I was looking for ^__^ Thanks

Reply:
My bad for my incorrect function